sdk/order-planner-service/tests/service/OrderPlannerService.spec.ts

```diff
@@ -0,0 +1,204 @@
+import { ChainFamilyMap, IUser } from '@summerfi/sdk-client'
+import { FlashloanProvider, Simulation, SimulationType } from '@summerfi/sdk-common/simulation'
+import { DeploymentIndex } from '@summerfi/deployment-utils'
+import { ISwapManager } from '@summerfi/swap-common/interfaces'
+import { Address, ChainInfo } from '@summerfi/sdk-common/common'
+import { IPositionsManager } from '@summerfi/sdk-common/orders'
+import {
+  MakerPaybackAction,
+  MakerWithdrawAction,
+  ProtocolPluginsRegistry,
+  SparkBorrowAction,
+  SparkDepositAction,
+} from '@summerfi/protocol-plugins'
+import { MakerPoolId } from '@summerfi/sdk-common/protocols'
+
+import { SetupDeployments } from '../utils/SetupDeployments'
+import { UserMock } from '../mocks/UserMock'
+import { SwapManagerMock } from '../mocks/SwapManagerMock'
+import { SkippableActionCall, decodeOpExecutorCalldata } from '../utils/OpExecutorDecoding'
+import { decodeActionCalldata } from '../utils/ActionDecoding'
+import { FlashloanAction, ReturnFundsAction } from '../../src/actions'
+import { getRefinanceSimulation } from '../utils/RefinanceSimulation/RefinanceSimulation'
+import { OrderPlannerService } from '../../src/implementation/OrderPlannerService'
+import { getMakerPosition } from '../utils/Maker/MakerSourcePosition'
+import { getSparkPosition } from '../utils/Spark/SparkTargetPosition'
+
+import assert from 'assert'
```

Comment:
can use jest assert instead for consistency, it's almost the same

Reply:
Hey, there is no assert in Jest. Let's talk when you are back